static/faq.html

```diff
@@ -291,6 +293,22 @@ <h2 id="default-connections">
                 </li>
             </ul>
 
+            <h2 id="cellular-tracking">
+                <a href="#cellular-tracking">What does GrapheneOS do about cellular tracking and
+                silent SMS?</a>
+            </h2>
+
+            <p>GrapheneOS always considers the network to be hostile. Merely getting a silent SMS is
+            not a good indicator of being targeted by your cell carrier, police or government
+            because <i>anyone on the cell network can send them</i>. Cellular triangulation will
+            happen regardless of whether or not SMS texts are being sent or recieved by the phone,
+            and they don't have to even be a silent SMS; it wouldn't be much different than
+            recieving an unsolicited spam SMS!</p>
+
+            <p>Activating airplane mode will switch the cell radio off entirely. While airplane mode
```

Comment:
Should clarify and state that it fully disables cellular radio transmit / receive capabilities. The baseband processor provides more than the cellular radio. On the currently supported devices, it provides both the Wi-Fi and cellular radios with separately sandboxed code for each.

Reply:
Woah, whew, good to know, I'll go and fix that...